aws/resource_aws_elastic_beanstalk_environment_test.go

```diff
@@ -1312,5 +1298,5 @@ resource "aws_elastic_beanstalk_environment" "default" {
 EOF
   }
 }
-`, randInt, randInt, randInt, randInt, randInt, randInt)
+`, appName, queueName, keyPairName, instanceProfileName, roleName, policyName, envName)
```

Comment:
Maybe we should consider text templates for complex ones like these? 😬

Reply:
Yeah, possibly, I'm up for further improvements, but I think it's more important to get tests stable at this point. 😅